### PR TITLE
Sampling docs changes

### PR DESCRIPTION
**Pull request by bhautikpip**
Fixes

https://github.com/aws-otel/aws-otel.github.io/issues/201

https://github.com/aws-otel/aws-otel.github.io/issues/202

### CODE REVIEW

File: src/docs/getting-started/java-sdk/trace-auto-instr.mdx [hunk 1]
```diff
@@ -60,7 +60,7 @@ For example, to set the random sampling rate for creating traces, you can set th
 
 The ADOT Java Auto-Instrumentation Agent can be configured to use [X-Ray remote sampling](https://docs.aws.amazon.com/xray/latest/devguide/xray-console-sampling.html)
 by setting the environment variable `OTEL_TRACES_SAMPLER=xray`. You will also need to [configure the OpenTelemetry collector](/docs/getting-started/remote-sampling)
-to allow the application to fetch sampling configuration.
+to allow the application to fetch sampling configuration. By setting `OTEL_TRACES_SAMPLER_ARG=endpoint=localhost:4000` environment variable you can change the endpoint the sampler talks with when getting sampling configuration from AWS X-Ray Console.
```
Comment (bhautikpip) [inline src/docs/getting-started/java-sdk/trace-auto-instr.mdx:63]: Confirm with @anuraaga if we're reading this env var for sampling args.

File: src/docs/getting-started/remote-sampling.mdx [hunk 2]
```diff
@@ -69,3 +69,15 @@ to export trace data and fetch sampling rules.
 without authentication. If not using any other authentication mechanism in front of the collector, it is recommended to
 configure network policy to only allow network connectivity to applications that should be allowed to access X-Ray with
 the above IAM permissions, for example using network firewall rules.**
+
+## OTel Remote Sampling Implementation Caveats
+
+Below mentioned table displays X-Ray sampling attributes and X-Ray equivalent OTel sampling attributes used by the OTel remote sampling implementation. Here's how you can [configure X-Ray sampling attributes](https://docs.aws.amazon.com/xray/latest/devguide/xray-console-sampling.html#xray-console-config) in AWS X-Ray Console. Below mentioned caveats to keep in mind when using OTel remote sampling implementation.
+
+1. Some of the X-Ray equivalent OTel sampling fields are instrumentation specific like `HTTP_TARGET`, `HTTP_URL`, `HTTP_METHOD` and `HTTP_HOST`. If instrumentation you're using in your application won't set them then remote sampling implementation would not have all these fields to match against in that case it would only use available fields like `SERVICE_NAME` and `CLOUD_PLATFORM`
```
Comment (anuraaga) [inline src/docs/getting-started/remote-sampling.mdx:77]: The format for OTel attributes looks like `http.target`

File: src/docs/getting-started/java-sdk/trace-auto-instr.mdx (hunk 1 above)
Comment (willarmiros) [reply src/docs/getting-started/java-sdk/trace-auto-instr.mdx:63]:
```suggestion
to allow the application to fetch sampling configuration. By default the sampler sends requests to `http://localhost:2000`. By setting `OTEL_TRACES_SAMPLER_ARG` environment variable you can change the endpoint the sampler talks with when getting sampling configuration from AWS X-Ray Console. For example setting `OTEL_TRACES_SAMPLER_ARG=endpoint=localhost:4000` would configure the sampler to communicate with `localhost:4000`.
```

File: src/docs/getting-started/java-sdk/trace-manual-instr.mdx [hunk 3]
```diff
@@ -290,7 +290,9 @@ Resource resource = Resource.builder()
 OpenTelemetrySdk.builder()
   .setTracerProvider(SdkTracerProvider.builder()
     .setResource(resource)
-    .setSampler(AwsXrayRemoteSampler.newBuilder(resource).build())
+    .setSampler(AwsXrayRemoteSampler.newBuilder(resource).setEndpoint("http://localhost:2000")
+                                                                      .setPollingInterval(Duration.ofSeconds(5))
```
[the remark this answers is not in the document]
Comment (bhautikpip) [reply src/docs/getting-started/java-sdk/trace-manual-instr.mdx:294]: yep. default value is 300 https://github.com/open-telemetry/opentelemetry-java-contrib/blob/b5d88e223c8c316b5d97474e5d2ff8fc4b428af1/aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSamplerBuilder.java#L46

[the remark this answers is not in the document]
Comment (willarmiros) [reply src/docs/getting-started/java-sdk/trace-manual-instr.mdx:299]: So should we use default value of 300 here as well? I feel like 5 seconds is too often, and wouldn't want customers to copy/paste this

[the remark this answers is not in the document]
Comment (bhautikpip) [reply src/docs/getting-started/java-sdk/trace-manual-instr.mdx:299]: Yeah makes sense! will change it.

**Review by willarmiros — APPROVED** [review]
LGTM, thanks!

**Review by alolita — APPROVED** [review]

**Pull request merged**